modules/sms_user/config/install/sms_user.settings.yml

```diff
@@ -1,6 +1,15 @@
 active_hours:
   status: false
   ranges: []
-registration_enabled: false
-allow_password: false
-new_account_message: ''
+account_registration:
+  all_unknown_numbers:
+    status: false
+    reply_message: ''
+  formatted:
+    status: false
+    incoming_messages: []
+    activation_email: true
+    reply:
+      status: false
+      message: ''
+      message_failure: ''
```

Comment:
The installed config you quoted has not been updated to match the schema. If you were to install the module at  that snapshot you would get invalid config. Schema is accurate.

It likely looks better when you're getting the variables, and looking at schema.

I am yet to find a better name for preformatted --- naming things is hard.

> 
> all_unknown_numbers --> register_unknown_numbers
> formatted --> register_preformatted_message

register_ is implied by the parent; account_registration.

> register_preformatted_message

_message doesnt describe contents, as there is much more than message inside.

> reply --> reply_message

I mean we have reply.status and reply.message when using get. so this is redundant.

**Some examples of get calls using proper schema:**

```
account_registration.all_unknown_numbers.status
account_registration.all_unknown_numbers.reply.status
account_registration.all_unknown_numbers.reply.message
account_registration.formatted.status
account_registration.formatted.incoming_messages.SEQUENCE
account_registration.formatted.activation_email
account_registration.formatted.reply.status
account_registration.formatted.reply.message
```